pkg/cmd/cli/cmd/set/env.go

```diff
@@ -273,14 +382,45 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				c.Env = updateEnv(c.Env, env, remove)
 
 				if list {
+					resolveErrors := map[string][]string{}
```

Comment:
looks fine, can you create something with several containers with several secret refs, and include the output of the following:
1. non-resolve call
2. resolve call run as editor
3. resolve call run as viewer